app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -93,16 +90,14 @@ export class ThemelessSubjectAreaDropdown extends React.Component {
   handleChange = selectedOptions => {
     if (selectedOptions.length <= 2) {
       this.setState({ selectedOptions, hasReachedMultiselectLimit: false })
-      if (this.props.onChange) this.props.onChange(selectedOptions)
+      this.props.onChange(selectedOptions)
     }
     if (selectedOptions.length >= 2) {
       this.setState({ hasReachedMultiselectLimit: true })
     }
   }
 
-  handleBlur = e => {
-    if (this.props.onBlur) this.props.onBlur(e)
-  }
+  handleBlur = e => this.props.onBlur(e)
```

Comment:
Is this line needed anymore?

Reply:
Yes

Without it, a blur event occurs on the input field - an element encapsulated within the Select component (from the `react-select` library) - but this isn't handled within the SubjectAreaDropdown component & therefore the ValidatedField component doesn't know to do anything on blur

Reply:
What I meant was: you wrote a function that calls another function with the same arguments. A better way to write this (in my opinion) would be:

```
...
onBlur: this.props.onBlur
...
```

Reply:
Now looking better at your code, just destructure it on line 104, along with `name` and other props. Also, sorry for being too vague if that's the case.

Reply:
I'm not sure that I understand. How would you "destructure" this? Where would I write `onBlur: this.props.onBlur`?

`handleBlur` is a function definition, not a variable. So even if I called the thing on the left & the thing on the right by the same name, I can't do something like:
`{ onBlur } = e => this.props`.

Besides this, there is a good reason for calling this `handleBlur` instead of `onBlur`. handleBlur is a method on SubjectAreaDropdown & onBlur is a prop. They serve different purposes and the naming conventions reflect that.

Reply:
Ah!!! Destructure [on this line](https://github.com/elifesciences/elife-xpub/pull/198/files#diff-1be53589f0d3af8629dbf22cdcb80cc6R99)

Beautiful, thanks @dryhten :+1: